README.md

```diff
@@ -34,6 +47,11 @@ For more detailed information about `mender-convert` please visit the
 
 ### Prepare image and configuration
 
+*Only 32bit-Images are supported at the moment!*
+
+The following steps give a quick start using Raspbian. For a more detailed guide,
+especially concerning version compatibilities, please visit [the the corresponding thread](https://hub.mender.io/t/raspberry-pi-3-model-b-b-raspbian/140) on the Mender Hub.
```

Comment:
Suggestion: This section is a bit confusing (it already was before you edited it). It makes the assumption that you are using Raspberry Pi, but nothing before this section indicates this. Should we perhaps put this under its own Raspberry Pi header?

Reply:
I agree that it is suboptimal. My main concern is that we're seeing a split of source of truth here. The current README suggests that the various threads on the Hub are valid information on supported boards, which I find problematic. I wanted to get something out quickly to avoid further confusion in the mean time - but my personal favortie would be to list the actively supported boards (via included configurations) in the README, and refer to the Hub only as "additional" information exchange.